src/components/common/ProductCard/index.js

```diff
@@ -2,21 +2,30 @@ import axios from "axios";
 import "./index.css";
 import { useState } from "react";
 import { useWishList } from "../../../context/wishlistContext";
-import { WISHLIST_STATUS_CLASS } from "../../../reducer/wishlist/wishlistConstants";
+import { deleteCartHandler, addToCart } from "../../../utils/handler";
 
-const ProductCard = ({ item, clickHandler, from }) => {
-  const [activeClass, setActiveClass] = useState("wishlist-icon")
-  const {state, dispatch} = useWishList()
+const ProductCard = ({ item, clickHandler, from, cartHandler }) => {
+  const [activeClass, setActiveClass] = useState("wishlist-icon");
+  const { state, dispatch } = useWishList();
 
   const btnHandler = () => {
     // dispatch({type: WISHLIST_STATUS_CLASS})
-    clickHandler({ item })
-    if(activeClass ==="wishlist-icon") {
-      setActiveClass("wishlist-active")
+    clickHandler({ item });
+    if (activeClass === "wishlist-icon") {
```

Comment:
use ternary operator for single if else condition,if there are multiple then make a object with key as active class and value as classname